### PR TITLE
Reload selected schedules on update

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Reload selected schedules on update

Configure selected count to enable button
Close tooltip on back button

[stage-19]

## Motivation and Context
More fixes and improvements for the release 

## How Has This Been Tested?
Tested changes locally.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu I found a small problem. When I load a Presentation that is not assigned to a Schedule, when I add its first schedule and click Select, it is not reloading the selectedSchedules list (i.e. count is not updated, and my assigned schedule does no appear in schedule selector if I reopen it).

Also, I adde another improvement to the list: 'Show spinner in Schedules component while loading selected Schedules'. Thanks!

File: web/scripts/schedules/services/svc-schedule-selector-factory.js [hunk 1]
```diff
@@ -71,64 +73,86 @@ angular.module('risevision.schedules.services')
...
-          return;
+          return $q.resolve();
         }
 
         var playlistItem = playlistFactory.newPresentationItem(templateEditorFactory.presentation);
 
         return playlistFactory.initPlayUntilDone(playlistItem, templateEditorFactory.presentation, true)
           .then(function () {
-            schedule.addPresentation(scheduleIds, JSON.stringify(playlistItem));
+            return schedule.addPresentation(scheduleIds, JSON.stringify(playlistItem));
```
Comment (alex-deaconu) [inline web/scripts/schedules/services/svc-schedule-selector-factory.js:97]: @ezequielc added missing return here to fix the issue with the list not refreshing. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc good catch! Fixed see inline comment. Also reverted config changes as I made the version default on rvacore-test. Thanks.

**Pull request merged**